src/app/[locale]/incident/add/components/AdditionalInformationPage.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+
+import { useTranslations } from 'next-intl'
+import { useFormStore } from '@/store/form_store'
+import { Heading, HeadingGroup, PreHeading } from '@/components'
+import { IncidentQuestionsLocationForm } from '@/app/[locale]/incident/add/components/IncidentQuestionsLocationForm'
+
+const currentStep = 2
+const maxStep = 4
+
+export const AdditionalInformationPage = () => {
+  const t = useTranslations('describe-add')
+  const tGeneral = useTranslations('general.describe_form')
+  const { loaded } = useFormStore()
+
+  // TODO: implement nice loading state, for if loaded if false. Also create a loading state for Stepper store.
+  if (loaded) {
+    return (
+      <div className="flex flex-col gap-4">
+        <HeadingGroup>
+          <Heading level={1}>{t('heading')}</Heading>
+          <PreHeading>
+            {tGeneral('pre-heading', { current: currentStep, max: maxStep })}
+          </PreHeading>
+        </HeadingGroup>
+        <IncidentQuestionsLocationForm />
+      </div>
+    )
+  }
+}
```

Comment:
Heeft deze pagina ook nog een title nodig?

Reply:
Nee elk *Page component in een components directory heeft dat niet nodig. Ik heb dit zo opgezet omdat de componenten zelf client componenten moeten zijn ivm afhankelijkheid van de globale store en de pagina's (page.tsx) files server components moeten zijn ivm de generateMetadata functie.